modules/24_trade/se_trade/datainput.gms

```diff
@@ -180,16 +180,34 @@ $ifthen.import_h2_EU "%cm_Ger_Pol%" == "ensec"
   p24_seTradeCapacity(t,regi,regi2,entySe)$(t.val eq 2030) = p24_seTrade_Quantity(regi,regi2,entySe)*0.2;
 $endif.import_h2_EU
 
+*** parameters used in cm_import_EU scenarios nzero, nzero_bio and high_bio
+
+*** calculating the share of FE demand per carrier for each region in each region group
+p24_seAggReference(ttot,regi,seAgg) = sum(enty$seAgg2se(seAgg,enty), sum(se2fe(enty,enty2,te), pm_prodFEReference(ttot,regi,enty,enty2,te)));
+p24_FEShareInRegion(ttot,ext_regi,regi,seAgg)$(regi_group(ext_regi,regi) and p24_seAggReference(ttot,regi,seAgg)) = p24_seAggReference(ttot,regi,seAgg) / sum(regi2$regi_group(ext_regi,regi2), p24_seAggReference(ttot,regi2,seAgg));
+
+execute_load "input_ref.gdx", p24_demFeForEsReference = vm_demFeForEs.l;
+execute_load "input_ref.gdx", p24_demFeIndSubReference = o37_demFeIndSub;
+
+*** calculating share of FE aviation and chemicals demand at each region group
```

Comment:
I would clarify to "calculate regional share of each region in the total of region group `ext_regi` with respect to chemicals + aviation liquids FE demand"